libs/components/forms/src/lib/modules/checkbox/checkbox.component.spec.ts

```diff
@@ -510,6 +510,7 @@ describe('Checkbox component', () => {
       );
       const labelTextSpy = spyOn(labelTextRequiredSvc, 'validateLabelText');
       fixture.detectChanges();
+      await fixture.whenStable();
```

Comment:
Now that you're using a simple setter in the host directive, you can probably safely remove these async changes to the tests.